snuba/snuba_migrations/replays/0011_add_is_dead_rage.py

```diff
@@ -0,0 +1,57 @@
+from typing import Iterator, Sequence, Tuple
+
+from snuba.clickhouse.columns import Column, UInt
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+
+new_columns: Sequence[Tuple[Column[Modifiers], str]] = [
+    (Column("click_is_dead", UInt(8)), "click_title"),
+    (Column("click_is_rage", UInt(8)), "click_is_dead"),
+]
+
+
+class Migration(migration.ClickhouseNodeMigration):
+    blocking = False
+
+    def forwards_ops(self) -> Sequence[operations.SqlOperation]:
+        return list(forward_columns_iter())
+
+    def backwards_ops(self) -> Sequence[operations.SqlOperation]:
+        return list(backward_columns_iter())
+
+
+def forward_columns_iter() -> Iterator[operations.SqlOperation]:
```

Comment:
Why is this a separate function that `yield`s ? Why not a simple list?

Reply:
@evanh Its just the pattern I've established previously and am carrying over.  I'm aware that I can append to and return a list but yielding works just as well for me.  The two are functionally equivalent but yield requires less indentation and (for me) requires less thought when reading and writing migration operations.

Reply:
Right, this is a stylistic nit: this is needlessly complicated. Simple is better than complex and flat is better than nested.